.github/workflows/individual-image-scanner-quay.yaml

```diff
@@ -105,7 +113,7 @@ jobs:
   check-results:
     runs-on: ubuntu-latest
     needs: scans
-    if: always()
+    if: ${{ github.event.workflow_run.conclusion == 'success' }}
```

Comment:
The reason for having always() in if is that we need to do the check irrespective of what happens with the previous steps for any one of the image. Eg: access-setup might go through but cluster-setup may not run. This may cause the action to exit without running check-results step.

Reply:
The `scans` job would `succeed` even if one of the steps is executed successfully based on path filtering. We will only get a `failure` when one of the steps which were to be executed based on path filtering doesn't go through. Is that correct? In that case, why would we need to do `check-results` because the Workflow would result in an error?

Also, I do see we won't need an `if` statement because `check-results` already has an `needs: scans` statement so it wouldn't run if `scans` job is not there or skipped!

Reply:
I still don't see any changes. Commit not pushed?

Reply:
I understand that we wanted to revert the `if` statement in `check-results` steps. The change is pushed 😓